bundle/src/main/java/com/cognifide/cq/cqsm/foundation/actions/destroyuser/DestroyUser.java

```diff
@@ -0,0 +1,67 @@
+package com.cognifide.cq.cqsm.foundation.actions.destroyuser;
+
+import java.util.Collections;
+
+import javax.jcr.RepositoryException;
+
+import org.apache.jackrabbit.api.security.user.User;
+
+import com.cognifide.cq.cqsm.api.actions.Action;
+import com.cognifide.cq.cqsm.api.actions.ActionResult;
+import com.cognifide.cq.cqsm.api.exceptions.ActionExecutionException;
+import com.cognifide.cq.cqsm.api.executors.Context;
+import com.cognifide.cq.cqsm.api.utils.AuthorizablesUtils;
+import com.cognifide.cq.cqsm.core.utils.MessagingUtils;
+import com.cognifide.cq.cqsm.foundation.actions.CompositeActionResult;
+import com.cognifide.cq.cqsm.foundation.actions.purge.Purge;
+import com.cognifide.cq.cqsm.foundation.actions.removeuser.RemoveUser;
+
+public class DestroyUser implements Action {
+
+	private final Action purge;
+
+	private final Action remove;
+
+	private final String userId;
+
+	public DestroyUser(String userId) {
+		this.purge = new Purge("/");
+		this.userId = userId;

```

Comment:
I would also love to have the authorizable removed from all groups he's member of